### DISCUSSION TITLE
LDAP sync interval

### DISCUSSION THREAD

**Discussion by sskokorin** [goauthentik/authentik#4783] [Q&A] [answered]
Is there a way to reduce the amount of time between LDAP synchronization attempts?
The documentation provides the [AUTHENTIK_LDAP__TASK_TIMEOUT_HOURS](https://goauthentik.io/docs/installation/configuration#authentik_ldap__task_timeout_hours) option. The value is specified in hours. By default, the value is 2.
What if I want 10 minutes?

**Comment by sevmonster** [comment]
I tried setting `AUTHENTIK_LDAP__TASK_TIMEOUT_HOURS=0.1667` (approx. 10 minutes) and it looks like it caused the events to fail to be rescheduled? Might want to open up an issue for this.

**Comment by kidhab** [comment, marked as answer]
`AUTHENTIK_LDAP__TASK_TIMEOUT_HOURS` is not meant to control the sync interval. It a timeout value for the sync task itself.

The LDAP sync runs every [two hours](https://github.com/goauthentik/authentik/blob/main/authentik/sources/ldap/settings.py#L9). You can manually run a LDAP sync via [API](https://github.com/goauthentik/authentik/issues/4712#issuecomment-1523061560) or via [docker|podman-compose](https://goauthentik.io/docs/troubleshooting/ldap_source).
